src/basics.md

```diff
@@ -1266,6 +1267,37 @@ fn main() {
 }
 ```
 
+[ex-convert-datetime-timestamp]: #ex-convert-datetime-timestamp
+<a name="ex-convert-datetime-timestamp"></a>
+## Converting date to unix timestamp and vice versa
+[![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
+
+Converts a date given by [`NaiveDate::from_ymd`] and [`NaiveTime::from_hms`]
+to the number of seconds since January 1, 1970 0:00:00 UTC using [`NaiveDateTime::timestamp`].
+Then it calculates what was the date after one billion seconds
+since January 1, 1970 0:00:00 UTC, using [`NaiveDateTime::from_timestamp`].
+
+```rust
+extern crate chrono;
+
+use chrono::{NaiveDate, NaiveDateTime};
+
+fn main() {
+    let date_time: NaiveDateTime = NaiveDate::from_ymd(2017, 11, 12).and_hms(17, 33, 44);
+    let seconds_since_date_time: i64 = date_time.timestamp();
```

Comment:
- type ascription is not required here.
- Also the variable name is incorrect. its seconds between UNIX Epoch and the date_time. Lets just drop the intermediate variable and call `timestamp` within the println body